bangazon-clumsy-penguins/WorkforceManagement
# Pp training program

# Training Program List, Create, Details

## Description

Training Program controller displaying list, details, and option to create

## Steps to Test

Click Training in the Nav bar and a list of training should be displayed

Click Details and the details of start and end date and max occupancy should be displayed

Click Create new option and a form with name, dates, and occupancy should be displayed

warning messages should be displayed until all fields are filled out when clicking the create button

Upon successful create the new training should be displayed in the list of training

##### Link to Feature Ticket
#9 #10 (10 partial details only)
